archspec/cpu/microarchitecture.py

```diff
@@ -73,13 +73,14 @@ class Microarchitecture:
     #: Aliases for micro-architecture's features
     feature_aliases = FEATURE_ALIASES
 
-    def __init__(self, name, parents, vendor, features, compilers, generation=0):
+    def __init__(self, name, parents, vendor, features, compilers, generation=0, cpuid=""):
```

Comment:
We should use `cpupart` everywhere, including when we read the JSON file. There is no `cpuid` filed in the JSON...

Reply:
Hi @alalazo , I agree that having cpupart in the Json and the cpuid here can be confusing. However, as I explained above, @dslarm and I would prefer, at this stage, to add the necessary fields only for the ARM vendor (CPU Implementer 0x41) and not have it for all ARM based uarchs. threrefore I thought using cpuid anyway was going to enforce the use of CPU Implementer. Can change and use just the cpupart.